app/javascript/packs/admin_invoices/content.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function timeToDecimal(time) {
+  const arr = time.split(':');
+  return parseFloat(parseInt(arr[0], 10) + '.' + parseInt((arr[1]/6)*10, 10));
+}
+
+function getBusinessDatesCount(startDate, endDate) {
+  let count = 0;
+  const curDate = startDate;
```

Comment:
`curDate`? is this `currentDate`? Don't try to simplify/shortify the vars. It is a bad smell